pytest_timeout.py

```diff
@@ -79,6 +85,15 @@ def pytest_addoption(parser):
         action="store_true",
         help=DISABLE_DEBUGGER_DETECTION_DESC,
     )
+    group.addoption(
+        "--suite-timeout",
+        action="store",
+        dest="suite_timeout",
+        default=None,
+        type=float,
+        metavar="minutes",
```

Comment:
`metavar` is usually upper case?

But more importantly, I don't know how I feel about minutes.  All other durations in pytest-timeout are seconds.  At least once there have been people we have wanted a more flexible duration specification that makes the unit clearer.  But that doesn't exist so I guess we're stuck with just floats (which, btw, was picked consciously originally to match the unit of the `time.time()` family of functions).  But mixing the unit of that float in different places does not seem great.

Would you mind using seconds here?  Or is the larger numbers really off-putting?

Reply:
Seconds is fine. 
Upper case metavar does look better. thanks.